clients/js/src/ChromaClient.ts

```diff
@@ -94,7 +99,12 @@ export class ChromaClient {
   }
 
   /** @ignore */
-  init(): Promise<void> {
+  async init(): Promise<void> {
```

Comment:
nit: might be slightly cleaner to combine the two checks into the single `this._initPromise`
also should probably mark init() as private, not sure why it wasn't already

Reply:
I believe there're some tests relying on it being public